app/views/thesis/process_theses.html.erb

```diff
@@ -240,11 +233,18 @@
 
     <div class="field-row layout-band layout-1q3q">
       <div class="col1q">
-        <strong>Thesis supervisor(s)*</strong>
+        <strong>Thesis supervisor(s) *</strong>
         <p class="hint">(just supervisors, not entire committee)</p>
       </div>
-      <%= f.simple_fields_for :advisors do |advisor| %>
-        <%= render 'advisor_fields', :f => advisor %>
+      <% if f.object.advisors.any? %>
+        <%= f.simple_fields_for :advisors do |advisor| %>
+          <%= render 'advisor_fields', :f => advisor %>
+        <% end %>
+      <% else %>
+        <div class="nested-fields col3q no-advisors">
+          <label class="field-label" for="advisors">Supervisor name *</label>
+          <input class="field field-text wide input-readonly" readonly type="text" value="There are no advisors associated with this thesis." name="advisors">
```

Comment:
I think this is because the input tag was missing an id attribute. This is fixed in the most recent commit, but I can't confirm because accesslint won't run again.

Reply:
I'm not seeing an error in either WAVE or ANDI on this field, so I'm inclined to agree that you've fixed the issue.